doc/api/test.md

```diff
@@ -1471,6 +1481,16 @@ Emitted when code coverage is enabled and all tests have completed.
 
 Emitted when [`context.diagnostic`][] is called.
 
+### Event: `'test:enqueue'`
+
+* `data` {Object}
+  * `file` {string|undefined} The path of the test file,
+    undefined if test is not ran through a file.
```

Comment:
What are you trying to say here? 

"will not run" in this (and above) perhaps?

Reply:
I think running within REPL is the main example of this. Il refine this in a follow-up